front/src/utils/hooks/useLogo.ts

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+
+import defaultLogo from 'assets/logo-color.svg';
+import defaultOsrdLogo from 'assets/logo-osrd-color-white.svg';
+import proudLogo from 'assets/proud-logo-color.svg';
+import proudOsrdLogo from 'assets/proud-logo-osrd-color-white.svg';
+import xmasLogo from 'assets/xmas-logo-color.svg';
+import xmasOsrdLogo from 'assets/xmas-logo-osrd-color-white.svg';
+
+const MONTH_VALUES = {
+  JUNE: 5,
+  DECEMBER: 11,
+};
+
+const useLogo = () => {
+  const [digitalTwin, setDigitalTwin] = useState<{ logo: string; logoWithName: string }>({
+    logo: defaultLogo,
+    logoWithName: defaultOsrdLogo,
+  });
+  const [stdcm, setStdcm] = useState<{ stdcmLogo?: string; stdcmPngLogo?: string }>({
+    stdcmLogo: undefined,
+    stdcmPngLogo: undefined,
```

Comment:
Maybe stdcmSimulationSheetLogo would be a better name ?
```suggestion
    stdcmSimulationSheetLogo: undefined,
```

Reply:
Done